src/node.h

```diff
@@ -124,31 +124,6 @@ NODE_EXTERN v8::Local<v8::Value> UVException(v8::Isolate* isolate,
                                              const char* path = nullptr,
                                              const char* dest = nullptr);
 
-NODE_DEPRECATED("Use ErrnoException(isolate, ...)",
```

Comment:
I am not sure whether should we add `semver-minor` label for removing this.

Reply:
It’s a breaking change, so it’s `semver-major`. I wouldn’t remove it in this PR.

Reply:
Removed.